crates/uv/tests/it/pip_debug.rs

```diff
@@ -0,0 +1,16 @@
+use crate::common::{TestContext, uv_snapshot};
+
+#[test]
+fn debug_warn() {
+    let context = TestContext::new("3.12");
+
+    uv_snapshot!(context.pip_debug(), @r"
+    success: false
+    exit_code: 2
+    ----- stdout -----
+
+    ----- stderr -----
+    warning: The `debug` subcommand is not present in `uv pip`
```

Comment:
We might want to match the message of the pip compat args?

https://github.com/astral-sh/uv/blob/2df06ebfbc2d4df90d47ef9a9e631ba9926712e0/crates/uv-cli/src/compat.rs#L338

so

> pip's `debug` command is unsupported

Reply:
Ah, I did not find those when I grepped.

What should be the suggested alternative? Just normal `pip debug`?

Reply:
I don't think we have a great alternative. I guess we could recommend `uvx pip debug`?